app/views/teachers/classrooms/supplemental_materials/_supplemental_material.html.slim

```diff
@@ -3,6 +3,7 @@
     h2 = supplemental_material.name
     = raw supplemental_material.markdown_content
 
+    .small-12
     = link_to "Edit", edit_supplemental_material_path(supplemental_material), class: "button"
```

Comment:
Same here. If you wanted the link to be inside the `.small-12` div you have to nest it one more level.